### PR TITLE
Updated Windows 10 Enterprise 2004 to the latest revision of the benchmark

### PR DESCRIPTION
**Pull request by MKletz**
- Updated resource based on v1.9.1 #146 

- Didn't bother with a CIS-CAT because the actual resource did not update. This revision appears to have just reduced the number of static corrections needed from typo fixes. The only difference in the resource schema was the line ending.

- Examples updates are due to the existing ones pre-dating plaster creating these.

- The static corrections file was sorted but is essentially the same content

### CODE REVIEW

File: src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1 [hunk 1]
```diff
@@ -3,25 +3,27 @@
 
 <#
     .DESCRIPTION
-    Applies CIS Level one benchmarks for Windows 10 build 2004 with no exclusions.
+    Applies CIS Level one benchmarks for Microsoft_Windows_10_Enterprise build 2004 the no exclusions.
```
Comment (edthedev) [inline src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1:6]: was this 'with' supposed to become 'the'?

**Comment by edthedev** [comment]
Approved with a question on a comment.

File: src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1 (hunk 1 above)
Comment (MKletz) [reply src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1:6]: Looks like the plaster template is wrong.

Comment (MKletz) [reply src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1]: latest push fixes that on all the examples. Plaster fix is on #157 which reset your approval there.

Comment (MKletz) [reply src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1]: I know there are under scores in the OS name also. I need to open an issue for it. The fix is rather tedious to impliment.

**Review by apetsche1 — APPROVED** [review]

File: src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1 (hunk 1 above)
Comment (edthedev) [reply src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1]: I figured the underscores were an artistic choice.

**Review by edthedev — APPROVED** [review]

**Pull request merged**